config/logit/alerts.d/flatline.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Suggest naming this file `.yaml.erb`

Reply:
I did originally, but then I have to strip it in the ruby and eugh. done now!